3.Tests/UI/Features/Item/CheckItem.feature

```diff
@@ -0,0 +1,12 @@
+Feature: Checking a Todo Item
+    As a logged user I want to check a todo item in a project
+    So that I can keep track of my tasks
+
+    Background:
+        Given the user is logged in
+
+    @Smoke @create.project.NewProject @create.item
+    Scenario: Checking an Item in a Project successfully
+        Given the user has selected the "NewProject" project
+        When I check the item
```

Comment:
Here should say "When the user checks the item"

Reply:
thank you, now it's solved